lms/djangoapps/teams/views.py

```diff
@@ -180,6 +186,12 @@ def get(self, request, course_id):
                 "staff": bool(has_access(user, 'staff', course_key)),
                 "teams": user_teams_data
             },
+            "has_open_teamset": bool(teamset_counts_by_type[TeamsetType.open.value]),
+            "has_public_managed_teamset": bool(teamset_counts_by_type[TeamsetType.public_managed.value]),
+            "has_managed_teamset": bool(
+                teamset_counts_by_type[TeamsetType.public_managed.value] +
+                teamset_counts_by_type[TeamsetType.private_managed.value]
+            ),
```

Comment:
Should we add a unit test that makes sure these new keys are actually included in the template's context?